Sources/SwiftParser/IncrementalParseTransition.swift

```diff
@@ -285,20 +152,20 @@ public struct IncrementalParseLookup {
     // text, e.g. because `private struct Foo {}` parses as a CodeBlockItem with
     // a StructDecl inside and `private struc Foo {}` parses as two
     // CodeBlockItems one for `private` and one for `struc Foo {}`
-    var nextLeafNodeLength: SourceLength = .zero
+    var nextLeafNodeLength = 0
```

Comment:
Is there any reason why we can’t use `SourceLength` anymore? It is pretty useful just to ensure everybody is talking about lengths in terms of UTF-8 bytes and we don’t accidentally sneak lengths in terms of Unicode grapheme clusters `String.count` in.

Reply:
I think that's something left when I hacked around. I've changed this to `SourceLength`.